README.md

```diff
@@ -76,13 +75,13 @@ Installing the library
 
 To install the library, once you have Go [all set up][2]:
 
-    $ go get -u github.com/colinmarc/hdfs
```

Comment:
fix for https://github.com/DavidAnson/markdownlint/blob/v0.8.1/doc/Rules.md#md014